website/views/shared/tasks/task.jade

```diff
@@ -14,4 +14,4 @@ li(id='task-{{::task.id}}',
 
     include ./edit/index
 
-  div(class='{{obj._id}}{{task.id}}-chart', ng-show='charts[obj._id+task.id]')
+    
```

Comment:
Good point! Not quite sure what was going on in my head when I removed that. I just made a push to bring it back.